paperspace/client.py

```diff
@@ -31,23 +31,26 @@ def get_path(self, url):
     def post(self, url, json=None, params=None):
         path = self.get_path(url)
         response = requests.post(path, json=json, params=params, headers=self.headers)
-        logger.debug("POST request sent to: {} with headers: {}".format(response.url, self.headers))
+        logger.debug("POST request sent to: {} \n\theaders: {}\n\tjson: {}\n\tparams: {}"
+                     .format(response.url, self.headers, json, params))
```

Comment:
last time we had discussion about using `format` in logger - how does it comply to this logger?

Reply:
nope, this has nothing to do with python logging module. logger is an adapter for click's  logger